nupic/frameworks/opf/clamodel.py

```diff
@@ -441,6 +443,21 @@ def _getSensorInputRecord(self, inputRecord):
                        sequenceReset=resetOut,
                        category=inputRecordCategory)
 
+  def _getClassifierInputRecord(self, inputRecord):
+    """
+    inputRecord - dict containing the input to the sensor
+
+    Return a 'ClassifierInput' object, which contains the mapped
+    bucket index for input Record
+    """
+    if self._predictedFieldName is not None and self._classifierInputEncoder is not None:
+      absoluteValue = inputRecord[self._predictedFieldName]
+      bucketIdx = self._classifierInputEncoder.getBucketIndices(absoluteValue)[0]
+    else:
+      absoluteValue = None
+      bucketIdx = None
```

Comment:
Can replace with:

    absoluteValue = None
    bucketIdx = None

    if self._predictedFieldName is not None and self._classifierInputEncoder is not None:
      absoluteValue = inputRecord[self._predictedFieldName]
      bucketIdx = self._classifierInputEncoder.getBucketIndices(absoluteValue)[0]